### DISCUSSION TITLE
Shutting down environment reports "crash"

### DISCUSSION THREAD

**Discussion by 1800joe** [noop-inc/documentation#113] [Bugs] [answered]
v 0.7.0

Using Node template -- start Env, then stop. Resulting in logs reporting "crash"

<img width="759" alt="Screenshot 2024-02-12 at 9 04 01 AM" src="https://github.com/noop-inc/documentation/assets/3111670/5220d26d-e8c0-4185-8a0a-49bac99a2330">

**Comment by derekwolpert** [comment, marked as answer]
This issue has been fixed with Desktop v0.8.1
